src/System.Text.Encodings.Web/src/System/Text/Encodings/Web/TextEncoder.cs

```diff
@@ -819,7 +819,7 @@ private byte[] GetAsciiEncoding(byte value)
 
         private static void ThrowArgumentException_MaxOutputCharsPerInputChar()
         {
-            throw new ArgumentException("Argument encoder does not implement MaxOutputCharsPerInputChar correctly.");
+            throw new ArgumentException(SR.ArgumentEncoderDoesNotImplementMaxOutputCharsPerInputChar);
```

Comment:
It's strange to me that this is throwing an ArgumentException, since it's not about any method argument passed by the caller and rather about the derived type's override of a particular property.  But at this point it'd be a breaking change to change the exception type, and it's not worth it.

Reply:
Could we absorb such a breaking change for 5.0? We had discussed the usage of this type (particularly instances of folks deriving from it), and it was quite low (almost 0).